templates/registry/registry-cm.yaml

```diff
@@ -36,7 +36,9 @@ data:
       {{- else if eq $type "gcs" }}
       gcs:
         bucket: {{ $storage.gcs.bucket }}
+        {{- if $storage.gcs.encodedkey }}
         keyfile: /etc/registry/gcs-key.json
```

Comment:
Seems the `keyfile` here is useless, the volume used in deployment is created from secret. If it is, please remove this line

Reply:
`keyfile ` path is only useless if you are using workload identity. This is why if you specify an encoded key, the `keyfile` is still set otherwise it would fail to work when using the encoded key.